app/views/report/index.html.haml

```diff
@@ -3,26 +3,30 @@
     .col-md-3
       = render partial: "admins/admin_bar"
     .col-md-9
-      %h1
-        = t('report.title')
       #reports
+        %h1
+          = t("report.title")
         - @reports.each do |r|
-          - username = User.find_by_id(r.user_id).username
-          .content
-            %span.text
-              = report_content(r.item_id, r.item_type)
-            %span
-              = raw t('report.reported_label', person: link_to(username, user_profile_path(username)))
-            %span
-              = t('report.reason_label', text: r.text)
-          .options.text-right
-            %span
-              = button_to t('report.review_link'), report_path(r.id, :type => r.item_type),
-                :class => "btn btn-info btn-small",
+          .panel.panel-default
+            - username = User.find_by_id(r.user_id).username
+            .panel-heading
+              .reporter.pull-right
+                = raw t("report.reported_label", person: link_to(username, user_profile_path(username)))
+              .title
+                = report_content(r.item_id, r.item_type)
+            .panel-body
+              .reason
+                = t("report.reason_label", text: r.text)
+
+              = button_to t("report.reported_user_details"),
+                user_search_path(admins_controller_user_search: {guid: get_reported_guid(r.item_id,
+                r.item_type)}), class: "btn pull-left btn-info btn-small",
+                method: :post
+              = button_to t("report.review_link"), report_path(r.id, type: r.item_type),
+                class: "btn pull-left btn-info btn-small",
                 method: :put
-            %span
-              = button_to t('report.delete_link'), report_path(r.id, :type => r.item_type),
-                :data => { :confirm => t('report.confirm_deletion') },
-                :class => "btn btn-danger btn-small",
+              = button_to t("report.delete_link"), report_path(r.id, type: r.item_type),
```

Comment:
Maybe we can even get rid of passing the type here, by looking it up in the report fetched via its id in the controller.